thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/controller.js

```diff
@@ -20,34 +20,48 @@ export default Controller.extend({
 
   /**
    * Set initial view values
+   * @method initialize
+   * @return {undefined}
    */
-  filterBy: 'All',
-  isGraphReady: false,
-  isTunePreviewActive: false,
-  isTuneSaveSuccess: false,
-  isTuneSaveFailure: false,
-  selectedSeverityOption: 'Percentage of Change',
-  selectedTunePattern: 'None',
-  customPercentChange: '30',
-  selectedTuneType: 'current',
-  customMttdChange: '5',
-  predefinedRanges: {},
-  today: moment(),
-  selectedSortMode: '',
-  sortColumnStartUp: false,
-  sortColumnScoreUp: false,
-  sortColumnChangeUp: false,
-  sortColumnResolutionUp: false,
+  initialize() {
+    this.setProperties({
+      filterBy: 'All',
+      isGraphReady: false,
+      isTunePreviewActive: false,
+      isTuneSaveSuccess: false,
+      isTuneSaveFailure: false,
+      selectedSeverityOption: 'Percentage of Change',
+      selectedTunePattern: 'None',
+      customPercentChange: '30',
+      selectedTuneType: 'current',
+      customMttdChange: '5',
+      predefinedRanges: {},
+      today: moment(),
+      selectedSortMode: '',
+      sortColumnStartUp: false,
+      sortColumnScoreUp: false,
+      sortColumnChangeUp: false,
+      sortColumnResolutionUp: false
+    });
+  },
 
   /**
    * Severity display options (power-select) and values
    */
-  severityMap: {
-    'Percentage of Change': 'weight',
-    'Absolute Value of Change': 'deviation',
-    'Site Wide Impact': 'site_wide_impact'
-  },
+  severityMap: computed('alertData', function() {
```

Comment:
Can the `toCalculateGlobalMetric` change while the reference o `alertData` doesn't change?
If yes, we should compute base on `alertData.toCalculateGlobalMetric`

Reply:
No, these are not expected to change.